engine/swarm/admin_guide.md

```diff
@@ -239,27 +252,40 @@ you demote or remove a manager
 ## Recover from disaster
 
 Swarm is resilient to failures and the swarm can recover from any number
-of temporary node failures (machine reboots or crash with restart).
+of temporary node failures (machine reboots or crash with restart) or other
+transient errors. However, a swarm cannot automatically recover if it loses a
+quorum.
+
+In a swarm of `N` managers, a quorum (a majority) of manager nodes must always
+be available. For example, in a swarm with 5 managers, a minimum of 3 must be
+operational and in communication with each other. In other words, the swarm can
+tolerate up to `(N-1)/2` permanent failures beyond which requests involving
+swarm management cannot be processed. These types of failures include data
+corruption or hardware failures.
 
-In a swarm of `N` managers, there must be a quorum of manager nodes greater than
-50% of the total number of managers (or `(N/2)+1`) in order for the swarm to
-process requests and remain available. This means the swarm can tolerate up to
-`(N-1)/2` permanent failures beyond which requests involving swarm management
-cannot be processed. These types of failures include data corruption or hardware
-failures.
+### Recovering from losing a quorum
+
+If you lose a quorum of managers, you cannot administer the swarm. Attempting
+to remove a node from a swarm which does not have a quorum of managers results
+in an error like the following:
```

Comment:
This is technically true but I'm not sure why we would explain what happens when you try to remove a node from a swarm that already has too few nodes to function. Is it just an example of how management features stop working?